README.md

```diff
@@ -30,6 +30,13 @@ find . -name "*_shpec.sh" | entr shpec
 [shpec's own tests](https://github.com/rylnd/shpec/tree/master/shpec/shpec_shpec.sh)
 are a great place to start. More detailed examples to come.
 
+### Mocking
```

Comment:
Indeed everything is visible on github, just another view of the issues, milestone and co.
However you need to add me as a Contributor on the project so I can access to milestone, and issues. (I had a look on waffle, and you can only create.contribute to a repo where you are part of. So I only see my fork, so no issues)